.github/workflows/fork-ci.yaml

```diff
@@ -14,24 +14,24 @@ jobs:
       REGISTRY_INTERNAL: nvcr.io/nvstaging/mellanox
       REGISTRY_PUBLIC: nvcr.io/nvidia/cloud-native
```

Comment:
The public repo for sriov NO images is:
`nvcr.io/nvidia/mellanox`

Reply:
fixed